test/mobile/test_lite_script_module.py

```diff
@@ -431,5 +445,109 @@ def forward(self, val: int, x, y, w):
         self.assertTrue('test_lite_script_module.py\", line {}'.format(lineno + 8) in error_message)
         self.assertTrue('top(FooTest5)' in error_message)
 
+
+class TestLiteScriptQuantizedModule(QuantizationTestCase):
+
+    def _compare_script_and_mobile(self,
+                                   model_class: Type[torch.nn.Module],
+                                   input: torch.Tensor,
+                                   **kwargs):
+        qengine = "qnnpack"
+        with override_quantized_engine(qengine):
+            qconfig = torch.quantization.get_default_qconfig(qengine)
+            model = model_class(**kwargs)
+            # model.qconfig = qconfig
+            # model = prepare(model)
+
+            # test_only_eval_fn(model, self.calib_data)
+            # model = convert(model)
+            model = quantize(model, test_only_eval_fn, [self.calib_data])
+
+            script_module = torch.jit.script(model)
+            script_module_result = script_module(input)
```

Comment:
Do we want to do jit.script save-load also to be consistent?

Reply:
I think `save` and `load` functions would be tested for consistency in pytorch-jit test bench. If you feel we should save and load script models, we can add them too. cc: @iseeyuan

Reply:
I think save and load are covered by _save_for_lite and _load_for_lite. Basically, _save_for_lite would call save, plus the mobile part.